pages/ksphere/kommander/1.1/tutorials/on-prem/index.md

```diff
@@ -97,10 +102,9 @@ spec:
 ---
 ```
 
-<p class="message--important"><strong>IMPORTANT: </strong>You must paste the above into the **cluster.yaml** field before you insert your `cluster.yaml`. For user, it is the ssh user tied to the keys pushed to your vms. This is the same user tied to the key uploaded in the cloud provider step.
-</p>
+<p class="message--important"><strong>IMPORTANT: </strong>You must paste the above <i>ClusterProvisioner</i> into the <i>cluster.yaml</i> field and then add the contents of your <code>cluster.yaml</code> after the three dashes. Replace <i>&lt;username&gt;</i> with the SSH user tied to the keys pushed to your machines. This is the same user tied to the key uploaded in the cloud provider step.</p>
```

Comment:
The use of italics for `ClusterProvisioner` and `cluster.yaml` seems non-conventional, why not keeping it as it was?

Reply:
This is HTML - it is broken when leaving as is.

<img width="916" alt="Screenshot 2020-07-23 at 14 37 16" src="https://user-images.githubusercontent.com/592445/88287039-0cf8b800-ccf2-11ea-81e5-597a965a7e21.png">